test/CMakeLists.txt

```diff
@@ -56,6 +56,7 @@ set(TEST_SOURCES
         TestProgramSettings.cc
         TestPvGenerator.cc
         TestRegion.cc
+        TestRegionImportExport.cc
```

Comment:
Not sure where this came from; it should be removed -- it's a duplicate of line 61. Bad merge?